impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -669,7 +679,12 @@ define(function() {
             yAxisSizeMax: 80, // shouldn't it be: maxHorizontalTextWidth ??
 
             // . dot
-            dot_lineWidth: function() { return this.finished(0); },
+            dot_lineWidth: function() {
+              // show nullShape
+              var v = this.getSize() != null ? 0 : this.delegate();
```

Comment:
Why not the same treatment of Bubble's `dot_lineWidth`?
I mean, either one has more code than it should or the other less than it should, no?